akka-remote/src/main/scala/akka/remote/artery/ArteryTransport.scala

```diff
@@ -757,6 +758,7 @@ private[remote] class ArteryTransport(_system: ExtendedActorSystem, _provider: R
         topLevelFREvents.loFreq(Transport_AeronErrorLogTaskStopped, NoMetaData)
       }
       if (aeron != null) aeron.close()
+      if (areonErrorLog != null) areonErrorLog.close()
```

Comment:
nope, I just removed it because segfaults ;) (previous PR had it) We will fix that scenario later.